### PR TITLE
First draft of trojan source post

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

**Comment by lattner** [comment]
Cool, let me know when it is ready for final review

**Comment by serge-sans-paille** [comment]
@lattner I think it is, the only problem I have is that one of the flag is still under review, for licensing reason (and thus it's going to take an unbound amount of time before we get a green light for it)

File: content/posts/2022-01-12-Trojan-Source.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+maintained by the Unicode consortium. In the case above, one would get a warning
+similar to
+
+```
+7:5: warning: 𝐟oo is confusable with foo [misc-homoglyph]
+```
+
+# Concluding Words
+
+Two of the above checks have been implemented in GCC as warnings within the
```
Comment (lattner) [inline content/posts/2022-01-12-Trojan-Source.md:104]: Do you mean "have been implemented in Clang as warnings"?

**Review by lattner — APPROVED** [review]

File: content/posts/2022-01-12-Trojan-Source.md (hunk 1 above)
Comment (serge-sans-paille) [reply content/posts/2022-01-12-Trojan-Source.md:104]: No, I wanted to point out the difference of approach between GCC and Clang there. I'll rephrase.

**Comment by lattner** [comment]
Nice, thank you!

**Pull request merged**